services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCAuthorizationDetailsProcessor.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2025 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.protocol.oid4vc.issuance;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import jakarta.ws.rs.core.MultivaluedMap;
+import jakarta.ws.rs.core.Response;
+import org.jboss.logging.Logger;
+import org.keycloak.events.Errors;
+import org.keycloak.events.EventBuilder;
+import org.keycloak.models.ClientSessionContext;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.UserSessionModel;
+import org.keycloak.protocol.oid4vc.model.AuthorizationDetail;
+import org.keycloak.protocol.oid4vc.model.AuthorizationDetailResponse;
+import org.keycloak.protocol.oid4vc.model.CredentialIssuer;
+import org.keycloak.protocol.oid4vc.model.SupportedCredentialConfiguration;
+import org.keycloak.services.CorsErrorResponseException;
+import org.keycloak.services.cors.Cors;
+import org.keycloak.util.JsonSerialization;
+import org.keycloak.protocol.oid4vc.model.Format;
+import static org.keycloak.protocol.oid4vc.model.Format.SUPPORTED_FORMATS;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+public class OID4VCAuthorizationDetailsProcessor {
+    private static final Logger logger = Logger.getLogger(OID4VCAuthorizationDetailsProcessor.class);
+    private final KeycloakSession session;
+    private final EventBuilder event;
+    private final MultivaluedMap<String, String> formParams;
+    private final Cors cors;
+
+    public static final String AUTHORIZATION_DETAILS_PARAM = "authorization_details";
```

Comment:
Note that this constant is also already defined here:
org.keycloak.protocol.oidc.OIDCLoginProtocol#AUTHORIZATION_DETAILS_PARAM

Perhaps the official parameters and response attributes should be added to the org.keycloak.OAuth2Constants.